index.html

```diff
@@ -49,14 +49,14 @@ <h3>Monero is secure.</h3>
                             <img src="/img/private-monero.png" alt="No surveillance" class="main-icon">
                         </div>
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12 why-text">
-                            <h3>Monero is private.</h3>
-                            <p>Monero uses ring signatures and ring confidential transactions to obfuscate the amounts, origins, and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions. </p>
+                            <h3>Monero is private</h3>
+                            <p>Monero uses ring signatures, ring confidential transactions and stealth addresses to obfuscate the origins, amounts and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions. </p>
```

Comment:
I don't know if we have a style guide that includes reference to the Oxford comma, but as the current dictator of this repo I demand it be used (or shown the guide). :)

I'm also not sure of the language around "ring signatures, ring confidential transactions..." as the former is subsumed by the latter.